lib/ansible/modules/remote_management/imc/imc_rest.py

```diff
@@ -296,10 +303,10 @@ def imc_response(module, rawoutput, rawinput=''):
     return result
 
 
-def logout(module, url, cookie, timeout):
+def logout(module, url, cookie, timeout, tls_insecure=False):
     ''' Perform a logout, if needed '''
     data = '<aaaLogout cookie="%s" inCookie="%s"/>' % (cookie, cookie)
-    resp, auth = fetch_url(module, url, data=data, method="POST", timeout=timeout)
+    resp, auth = fetch_url(module, url, data=data, method="POST", timeout=timeout, tls_insecure=tls_insecure)
```

Comment:
`fetch_url` does not have a `tls_insecure` parameter; it takes the value from `module.params['tls_insecure']` instead.

Reply:
Good catch, fixed.